packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -1,20 +1,32 @@
 // @flow
 import asyncMap from '@xen-orchestra/async-map'
+import createLogger from '@xen-orchestra/log'
 import defer from 'golike-defer'
 import { fromEvent, ignoreErrors } from 'promise-toolbox'
 
+import debounceWithKey from '../_pDebounceWithKey'
+import parseDuration from '../_parseDuration'
 import { type Xapi } from '../xapi'
 import {
   safeDateFormat,
+  serializeError,
   type SimpleIdPattern,
   unboxIdsFromPattern,
 } from '../utils'
 
 import { type Executor, type Job } from './jobs'
 import { type Schedule } from './scheduling'
 
+const log = createLogger('xo:xo-mixins:metadata-backups')
+
+const DEFAULT_DEBOUNCE_DELAY = 6e4
```

Comment:
No default value as a constant, the default value is in `config.toml`!

Reply:
How i can get the default value from the `config.toml`?